test/errors.js

```diff
@@ -1,7 +1,7 @@
 var chai = require('chai');
 var assert = chai.assert;
 
-var errors = require('../packages/web3-core-helpers/src/errors.js');
+var errors = require('../packages/web3-core-helpers').errors;
```

Comment:
I wonder if there's people importing things from `src/`. If there are, their projects will break.

I'm ok with considering everything under `src/` as private. But maybe a comment about this should be added to the next release notes.

Reply:
Yes, we should probably add a bigger "New Bundles" section.

Reply:
This can be addressed in a subsequent PR.